devtools/maven/src/main/java/io/quarkus/maven/CreateProjectMojo.java

```diff
@@ -183,7 +183,7 @@ public void execute() throws MojoExecutionException {
 
             BuildTool buildToolEnum;
             try {
-                buildToolEnum = BuildTool.valueOf(buildTool.toUpperCase());
+                buildToolEnum = BuildTool.valueOf(buildTool.toUpperCase().replace('-', '_'));
```

Comment:
what do you think of editing and using `BuildTool.findTool`:
```Java
    public static BuildTool findTool(String tool) {
        if ("GRADLE".equalsIgnoreCase(tool))
            return GRADLE;
        else if ("GRADLE_KOTLIN_DSL".equalsIgnoreCase(tool) || "gradle-kotlin-dsl".equalsIgnoreCase(tool))
            return GRADLE_KOTLIN_DSL;
        else if ("MAVEN".equalsIgnoreCase(tool))
            return MAVEN;
        else
            return null;
    }
```

then you would just have:
```Java
BuildTool.findTool(buildTool);
```

Reply:
Good idea. I will change this.